Example/AblyChatExample/ContentView.swift

```diff
@@ -3,7 +3,10 @@ import SwiftUI
 
 struct ContentView: View {
     /// Just used to check that we can successfully import and use the AblyChat library. TODO remove this once we start building the library
-    @State private var ablyChatClient = AblyChatClient()
+    @State private var ablyChatClient = DefaultChatClient(
```

Comment:
I’m not sure it makes sense to prefix anything in the SDK with "Ably", given that it's an SDK specifically designed for interacting with Ably.

As I see it, the options are:

1. don't put `Protocol` after any protocol name, meaning that the class that provides the default implementation of this protocol needs to have something that distinguishes it from the protocol — this is what I’ve done
2. do as in 1. but change the rules when there is a public class that implements the protocol, i.e. have class `ChatClient` and protocol `ChatClientProtocol`
3. put `Protocol` after all protocol names and remove the `Default` from the class names

I don’t like 2 because it's inconsistent. I’m indifferent whether we do 1 or 3.

Reply:
I don't like 3 (and 2). We work with protocols most of the time and don't care about implementation, so I don't think it's a good idea to have `Protocol` suffix everywhere.

Reply:
I'm keen to option 1 too, I just don't like the `Default` prefix much. "Default" means that there is going to be a specific one which is probably not going to happen, why call it "default" then? "Ably" suffix is also used where you should distinguish the type from the existing one, so it serves this purpose in this case too. But I'm not very happy with it either, because, as @lawrence-forooghian mentioned, it's obvious that it's an exclusively Ably chat client. We also use `Protocol` suffix in ably-cocoa for all major API types, so maybe to go with 3? WDYT @ttypic

Reply:
Honestly it's not something I really want to spend too much time thinking about now; it's very easy to change later if we’re still unhappy with it.